### ISSUE TITLE
Data Scientist JD @ Beagle Tech

### ISSUE THREAD

**Issue by wingchen** [beagle-ai/jobs#1]
# Company Introduction

We are a high-growth agriculture technology company, using AI and robotics to solve the labor shortage problem. Although we can search millions of Youtube videos today within one second and watch your favorite movie with one single click, farming today is still incredibly labor intensive. Our mission is to use AI and robotic technology to transform labor intensive farming. We have signed up the largest grape growers in the USA for paid trials this winter. Farmers are hungry for solutions, and we need help to build the product that will transform farming and impact millions of people.

You will work with a small but very elite team. Our team members come from Caltech, NASA JPL, Abbott Lab, and Western Digital. We have founded several tech companies before, and have Y Combinator alumni and industry veteran executives as our advisors. We have raised pre-seed investment from a renowned corporate incubator. We value creativity and responsibility, we manage the company with super transparency. If you are an AI expert who likes a high growth and high rewarding working environment, and want to see your product impact millions of people around the world, please talk with us.

# What we are solving

![antique-tractors](https://user-images.githubusercontent.com/798321/125570006-bd1c169c-b446-47b8-a1b8-a66a79336d1e.jpg)

We are taking the most modern technologies like computer vision, robotics, and deep learning, and applying them onto one of the oldest industries: agriculture. 

As you may know, the agriculture operations in the US are highly conducted with machines. However, to your surprise, most of those were designed before AI came about. There are a great number of possibilities to be implemented and productized in the next generation of AI-powered agriculture machines.

We are taking a lead in the revolution of AI-powered tractor automation.

# Who would be a good fit

If you like AI + Software + Robotics + Agriculture in a big market, this is the company for you. 

You will be working on building the entire data pipeline as well as the model pipeline to power the robotics part of the product. In other words, you will be working on the BRAIN part of the product.

An ideal candidate is an AI professional who has worked 2 to 3 years in the AI/computer vision industry and knows how to refine AI models starting from training data cleaning, preparation, and tagging, to various ways of model building and evaluation/benchmarking. 

# Basic Qualifications

- MS or PhD degree in Computer Science, Machine Learning, Physics, or related technical fields.
- Being able to understand research papers and convert the idea into a workable implementation.
- 2 years of educational / industrial experience in machine learning. We also consider personal projects as educational experience.
- Hands-on experience in Python programming as well as data science tool kits such as numpy, scipy, pandas, matplotlib.
- Experience in at least one of the modern Deep Learning frameworks: Tensorflow / Keras / PyTorch / Caffe / MXNet / Theano / CNTK.

# Bonus Qualifications

- Having domain knowledge in computer vision / signal processing fundamentals.
- 2 years industrial experience in machine learning / deep learning. Having a know-how on designing a scalable machine learning system.
- Proficiency in Python programming: able to write well-tested production code and familiar with the software development process of Python.

# Responsibilities

- Develop and maintain the existing end-to-end machine learning system: from data
collection, model training, model evaluation to the deployment of models on an
embedded system.
- Research and design experiments to improve model performance.
- Work on diversified requirements across different teams. Collaborating with embedded engineers, hardware engineers, data scientists, and CUSTOMERS (yes, you got it right) to provide machine learning solutions.

# Compensation

- 150K TWD monthly, 12 months yearly

# Location

- Taiwan, remote only

# Application

- email to: jobs@sailoi.com
